app/models/FileOperation.js

```diff
@@ -0,0 +1,23 @@
+// @flow
+import { computed } from "mobx";
+import BaseModal from "./BaseModel";
+import Collection from "./Collection";
+import User from "./User";
+
+class FileOperation extends BaseModal {
+  id: string;
+  state: string;
+  collection: ?Collection;
+  key: ?string;
+  size: number;
+  type: string;
+  user: User;
+  createdAt: string;
+
+  @computed
+  get sizeInMB(): string {
```

Comment:
As mentioned in comment, I think we need something slightly more sophisticated here that can deal with a wider range of sizes.

Reply:
Noted. Earlier I neglected this, thinking collections would generally be greater than MB in production.